eng/pipelines/runtime.yml

```diff
@@ -839,6 +839,8 @@ extends:
           platforms:
             - browser_wasm
             #- browser_wasm_win
+          jobParameters:
```

Comment:
this is probably not necessary anymore after https://github.com/dotnet/runtime/pull/104361